CHANGELOG

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Jul 13 07:40:52 UTC 2016 - jreidinger@suse.com
+
+- Do not crash for submenu with double quotes (bnc#986005)
+- filter out snapper menuentry ( using double quotes )
```

Comment:
question is if we should document reality or contract. For me change contract is to not crash on double quotes and filter out snapper menu entry. If someone report that their distro use double quotes, I am willing to change code to fix it and really only filter out snapper, still considering it only bug fix and not added feature. So filtering out double quotes is not set in stones nor feature on which someone should rely.